### DISCUSSION TITLE
Migrate up and down to specific migration to make the migration idempotent

### DISCUSSION THREAD

**Discussion by KeisukeYamashita** [SeaQL/sea-orm#1965] [Ideas]
## Motivation

In our organisation, migration is performed in Kubernetes jobs for auditing, elimination of gerrymandering and personal data protection reasons.

Jobs are handy but risky, especially when migrating down.

This is because, currently, only the `--num` flag is supported, which means that the database will migrate back as many times as it has been executed. Kubernetes can be configured so that a job only runs once, and all approach workarounds exist, but it would be nice to have support in the CLI if possible.

Operating is also challenging due to the lack of idempotency, even if you try to automate it like any other.
If such a flag were available, it would benefit practical operations.

Thank you for all your hard work!!

## Idea

Such a flag would be very welcome.

```console
$ sea-orm-cli migration up --to <NAME_OF_THE_MIGRATION>
$ sea-orm-cli migration down --to <NAME_OF_THE_MIGRATION>
```

**Comment by tyt2y3** [comment]
That's a great idea
